mbed_flasher/flashers/FlasherMbed.py

```diff
@@ -128,6 +128,7 @@ def runner(self, drive):
                 self.logger.debug("re-mount check timed out for %s", drive[0])
                 break
 
+    # Todo: the code here might need improvement
     # pylint: disable=too-many-branches
     def check_points_unchanged(self, target):
         """
```

Comment:
@VeliMattiLahtela I changed the logic here a bit, could you please help verify it doesn't break anything?